src/app/components/normal-component/header/header.component.html

```diff
@@ -33,6 +33,7 @@
                 <a class="navbar-link">{{ currentLanguage }}</a>
                 <div class="navbar-dropdown">
                     <a *ngFor="let language of availableLanguages"
+                       id="language_{{ language }}"
```

Comment:
so ... that's the reason it was not working with $localisable :D ? 🕋

Reply:
non ! c'est utilisé dans les tests